CHANGELOG.md

```diff
@@ -9,6 +9,7 @@
 - Fix `rails_best_practices` invocation from `bin/quality`, `config/rails_best_practices.yml` was ignored before.
 - Introduce [CSSComb](https://github.com/csscomb/csscomb.js). For more details see: https://github.com/fs/rails-base/pull/284
 - Add `Brewfile` to track application dependencies.
+- Add ability to close app from unauthorized viewers, see: https://github.com/fs/rails-base/pull/302/files
```

Comment:
please add it to "Unreleased" section. Time traveling is prohibited on our planet.